elasticsearch_metrics/management/commands/sync_metrics.py

```diff
@@ -1,12 +1,37 @@
-from django.core.management.base import BaseCommand
+from django.core.management.base import BaseCommand, CommandError
+
+from elasticsearch_metrics.registry import registry
 
 
 class Command(BaseCommand):
     help = "Ensures index templates exist for all Metric classes."
 
     def add_arguments(self, parser):
-        pass
+        parser.add_argument(
+            "app_label",
+            nargs="?",
+            help="App label of an application to synchronize the state.",
+        )
+        # TODO: "using"
 
     def handle(self, *args, **options):
-        # TODO
+        if options["app_label"]:
+            if options["app_label"] not in registry.all_metrics:
+                raise CommandError(
+                    "No metrics found for app '{}'".format(options["app_label"])
+                )
+            app_labels = [options["app_label"]]
+        else:
+            app_labels = registry.all_metrics.keys()
+        for app_label in app_labels:
```

Comment:
Seems straightforward enough...building on your earlier PR's, we loop through each app, and get all of the Metric classes.  Then for each Metric, we create the index template.